tests/test_output.py

```
@@ -0,0 +1,159 @@
+        )
+        self.assertEqual(sum(ind), 0)
+
+    def test_dump_chemical(self):
+        test_folder = os.path.join(self.static_folder, "dump_chemical")
+        structure_ni = bulk("Ni", cubic=True)
+        structure_al = bulk("Ni", cubic=True)
+        structure_all = bulk("Ni", cubic=True)
+        structure_ni.set_chemical_symbols(["H", "Ni", "Ni", "Ni"])
+        structure_al.set_chemical_symbols(["H", "Al", "Al", "Al"])
+        structure_all.set_chemical_symbols(["Al", "Al", "Ni", "H"])
+        for l, s, ind in zip(
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)